client/src/components/Workflow/Run/WorkflowRunForm.vue

```diff
@@ -39,9 +39,8 @@
                     <WorkflowRunToolStep
                         v-if="step.step_type == 'tool'"
                         :model="step"
-                        :step-data="stepData"
+                        :replace-params="getReplaceParams(step.inputs)"
```

Comment:
I don't think this is reactive anymore, you sure that can't become an issue ?

Reply:
It should be reactive, I created a small JSFiddle to demonstrate this behavior: https://jsfiddle.net/kjq38g04/.

Reply:
@mvdbeek that is correct. And you are also right with suggesting that computed values are slightly more efficient, however the answers also state that you would need a method if you need to parse a value to generate the computed result.